packages/framework/ai-collab/README.md

```diff
@@ -238,10 +239,166 @@ To see detailed information about each event, please read their cooresponding [t
 
 
 ### Using Trace Id's
+
 Debug Events in ai-collab have two different types of trace id's:
 - `traceId`: This field exists on all debug events and can be used to correlate all debug events that happened in a single execution of `aiCollab()`. Sorting the events by timestamp will show the proper chronological order of the events. Note that the events should already be emitted in chronological order.
 - `eventFlowTraceId`: this field exists on all `EventFlowDebugEvents` and can be used to correlate all events from a particular event flow. Additionally all LLM api call events will contain the `eventFlowTraceId` field as well as a `triggeringEventFlowName` so you can link LLM API calls to a particular event flow.
 
+## Edit Difference Visualizations
```

Comment:
I don't think "visualizations" is appropriate either. I think this can just be "Edit Differences" or "Edit Diffs"